src/ui/components/SecondaryToolbox/index.tsx

```diff
@@ -15,6 +14,8 @@ import { UIState } from "ui/state";
 import { PanelName } from "ui/state/app";
 import { isDemo } from "ui/utils/environment";
 
+const InspectorApp = React.lazy(() => import("devtools/client/inspector/components/App"));
```

Comment:
Is the inspector particularly heavyweight to lazy load it?

Reply:
No, but it's currently disabled by default so I thought it makes sense not to ship it to every user. But more importantly I used this to verify that it's relatively simple with the new setup organization to make a panel lazy-loadable.